frontend/src/lib/components/headingGroup/HeadingGroup.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import {_} from 'svelte-i18n';
+  import type {HeadingGroupProps} from './HeadingGroup.type';
+
+  type $$Props = HeadingGroupProps;
+
+  $$restProps['role'] ??= 'group';
+  $$restProps['aria-roledescription'] ??= $_('aria.headingGroup');
+</script>
+
+<!--
+@component
+Used to group the page's main headings, such as a pre-title and the
+main title.
+
+### Properties
+
+- `aria-roledescription`: The Aria role description of the `<hgroup>` 
+  element representing the pre-title.
+  @default $_('aria.headingGroup')
+- `role`: The Aria role of the `<hgroup>` element.
+  @default 'group'
+- Any valid attributes of a `<hgroup>` element.
+
+### Slots
+
+- default: the contents of the heading group.
+
+### Usage
+
+```tsx
+<HeadingGroup>
+  <PreTitle class="text-accent">Environment</PreTitle>
+  <h1>The Question About Environment</h1>
```

Comment:
I'll add it here, but in the other components' examples, I've held to a line not using translation strings, because they would get so cluttered with those and IMO making the examples less clear.